src/cache/precomputed_quantities.jl

```diff
@@ -441,27 +436,59 @@ NVTX.@annotate function set_precomputed_quantities!(Y, p, t)
     (; ᶜwₜqₜ, ᶜwₕhₜ) = p.precomputed
     @. ᶜwₜqₜ = Geometry.WVector(0)
     @. ᶜwₕhₜ = Geometry.WVector(0)
-    #
-    # TODO - uncomment in the next PR. Right now for the purpose of testing
-    # we want to merge with 0 sedimentation and precipitation
-    #
     if moisture_model isa NonEquilMoistModel
         set_sedimentation_precomputed_quantities!(Y, p, t)
-        #    (; ᶜwₗ, ᶜwᵢ) = p.precomputed
-        #    @. ᶜwₜqₜ += Geometry.WVector(ᶜwₗ * Y.c.ρq_liq + ᶜwᵢ * Y.c.ρq_ice) / Y.c.ρ
-        #    @. ᶜwₕhₜ += Geometry.WVector(
-        #        ᶜwₗ * Y.c.ρq_liq * (TD.internal_energy_liquid(thermo_params, ᶜts) + ᶜΦ + norm_sqr(Geometry.UVWVector(0, 0, -ᶜwₗ) + Geometry.UVWVector(ᶜu))/2) +
-        #        ᶜwᵢ * Y.c.ρq_ice * (TD.internal_energy_ice(thermo_params, ᶜts)    + ᶜΦ + norm_sqr(Geometry.UVWVector(0, 0, -ᶜwᵢ) + Geometry.UVWVector(ᶜu))/2)
-        #    ) / Y.c.ρ
+        (; ᶜwₗ, ᶜwᵢ) = p.precomputed
+        @. ᶜwₜqₜ +=
+            Geometry.WVector(ᶜwₗ * Y.c.ρq_liq + ᶜwᵢ * Y.c.ρq_ice) / Y.c.ρ
+        @. ᶜwₕhₜ +=
+            Geometry.WVector(
+                ᶜwₗ *
+                Y.c.ρq_liq *
+                (
+                    TD.internal_energy_liquid(thermo_params, ᶜts) +
+                    ᶜΦ +
+                    norm_sqr(
+                        Geometry.UVWVector(0, 0, ᶜwₗ) + Geometry.UVWVector(ᶜu),
```

Comment:
```suggestion
                        Geometry.UVWVector(0, 0, -(ᶜwₗ)) + Geometry.UVWVector(ᶜu),
```
If this doesn't work, let's ask Dennis.